contrib/firewalld/template.py

```diff
@@ -0,0 +1,60 @@
+pkgname = "firewalld"
+pkgver = "2.1.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--disable-systemd"]
+configure_gen = ["./autogen.sh"]
+make_cmd = "gmake"
+# icons won't install unless we do this...
+make_dir = "."
+hostmakedepends = [
+    "automake",
+    "gmake",
+    "gtk-doc-tools",
+    "intltool",
+    "libxml2-progs",
+    "pkgconf",
+    "python",
+]
+makedepends = [
+    "gettext-devel",
+    "glib-devel",
+]
+depends = [
+    "python-dbus",
+    "python-gobject",
+    "python-nftables",
+]
+pkgdesc = "Stateful zone-based firewall daemon with D-Bus interface"
+maintainer = "miko <mikoxyzzz@gmail.com>"
+license = "GPL-2.0-or-later"
+url = "https://firewalld.org"
+source = f"https://github.com/firewalld/firewalld/releases/download/v{pkgver}/firewalld-{pkgver}.tar.bz2"
+sha256 = "22e3c700c2e4867796e9c22cf929cba4b2c143c8884240cfb9f3ef731366ff71"
+# tests don't work in our build env
+options = ["!check"]
+
+
+def do_prepare(self):
+    self.rm("src/icons/*/*/firewall-applet*", recursive=True, glob=True)
+
+
+def post_install(self):
+    from cbuild.util import python
+
+    python.precompile(self, "usr/lib")
+    self.install_service(self.files_path / "firewalld")
+
+
+@subpackage("firewall-config")
+def _config(self):
+    self.pkgdesc = "GTK-based configuration utility for firewalld"
+    self.depends = ["gtk+3"]
```

Comment:
doesn't this depend on the parent package too (+ `f"{pkgname}={pkgver}-r{pkgrel}"`)

Reply:
true!